feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableItemDetailSummaryCard.kt

```diff
@@ -107,32 +136,128 @@ fun TimetableItemDetailSummaryCard(
 }
 
 @Composable
-private fun SummaryCardRow(
+private fun SummaryCardText(
     imageVector: ImageVector,
     contentDescription: String,
     title: String,
     description: String,
     modifier: Modifier = Modifier,
 ) {
-    Row(
-        modifier = modifier,
-        verticalAlignment = Alignment.CenterVertically,
-    ) {
-        Icon(imageVector = imageVector, contentDescription = contentDescription, tint = LocalRoomTheme.current.primaryColor)
-        Spacer(Modifier.width(8.dp))
-        Text(
-            text = title,
-            style = MaterialTheme.typography.titleSmall,
-            color = LocalRoomTheme.current.primaryColor,
-        )
-        Spacer(Modifier.width(8.dp))
-        Text(
-            text = description,
-            style = MaterialTheme.typography.bodyMedium,
+    val iconInlineContentId = "icon"
+    val spacer8dpInlineContentId = "spacer8dp"
+    val spacer12dpInlineContentId = "spacer12dp"
+
+    val annotatedString = createAnnotatedString(
+        title = title,
+        description = description,
+        iconInlineContentId = iconInlineContentId,
+        spacer8dpInlineContentId = spacer8dpInlineContentId,
+        spacer12dpInlineContentId = spacer12dpInlineContentId,
+    )
+
+    val inlineContent = createInlineContent(
+        imageVector = imageVector,
+        contentDescription = contentDescription,
+        iconInlineContentId = iconInlineContentId,
+        spacer8dpInlineContentId = spacer8dpInlineContentId,
+        spacer12dpInlineContentId = spacer12dpInlineContentId,
+    )
+
+    // If multiple Texts are placed, they will not break lines properly when the font scale is large.
+    // Therefore, AnnotatedString is used for implementation.
+    Text(
+        text = annotatedString,
+        inlineContent = inlineContent,
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentWidth(Alignment.Start),
+    )
+}
+
+@Composable
+private fun createAnnotatedString(
```

Comment:
This is a private method but It is a bit generic name. How about naming like this?

```suggestion
private fun createSummaryCardTextAnnotatedString(
```